diff --git a/Makefile b/Makefile
index 09944da..ca9d676 100644
--- a/Makefile
+++ b/Makefile
@@ -196,10 +196,10 @@ endif
 
 ifeq ($(PLATFORM),PC)
 # Linux 2.6
-LINUX_SRC = /lib/modules/$(shell uname -r)/build
+LINUX_SRC = ${KBASE}/build
 # Linux 2.4 Change to your local setting
 #LINUX_SRC = /usr/src/linux-2.4
-LINUX_SRC_MODULE = /lib/modules/$(shell uname -r)/kernel/drivers/net/wireless/
+LINUX_SRC_MODULE = ${KBASE}/kernel/drivers/net/wireless/
 CROSS_COMPILE = 
 endif
